components/tidb_query/src/codec/chunk/column.rs

```diff
@@ -258,6 +306,50 @@ impl Column {
         self.finish_append_fixed()
     }
 
+    pub fn append_f32_datum(&mut self, mut raw_datum: &[u8]) -> Result<()> {
```

Comment:
Please help add `#[inline]` annotations for all `get_xxx`, `append_xxx` for u64, i64, f64, f32, bytes.